### DISCUSSION TITLE
無法辨識剩餘體力

### DISCUSSION THREAD

**Discussion by Angrolo** [MaaAssistantArknights/MaaAssistantArknights#8422] [General]
我更新了最新bluestacks5版本 mma也更新最新版本，但還是無法刷理智其他都正常。

**Comment by status102** [comment]
带log提issue

**Reply by Angrolo** [reply]
我要怎麼給你看這問題 截圖?

**Reply by status102** [reply]
> 我要怎麼給你看這問題 截圖?

带log提issue，这里是discuss

**Reply by ABA2396** [reply]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose

**Reply by Angrolo** [reply]
請問紀錄檔案我在哪裡可以找到?

**Reply by ABA2396** [reply]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/99072975/af80af32-49db-426c-94fb-aef2d4c36811)

**Reply by Angrolo** [reply]
了解! 謝謝

**Reply by Angrolo** [reply]
解決了!後來發現是日服介面UI有被調過去maa官網抓取新圖檔替換就辨識錯的部分就可以了
